Makefile

```diff
@@ -86,6 +87,16 @@ else
 			 cd vendor/github.com/celo-org/bls-zexe/bls && cargo build --release --target=x86_64-linux-android --lib
 endif
 
+bls-zexe-ios:
+ifeq ("$(RUSTUP_exists)","")
+	$(error "No rustup in PATH, consult https://github.com/celo-org/celo-monorepo/blob/master/SETUP.md")
+else ifeq ("$(CARGO_LIPO_exists)","")
+	$(error "No cargo lipo in PATH, run cargo install cargo-lipo")
```

Comment:
How about installing cargo-lipo for the user here?

Reply:
Good idea! I'll do that.